app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -252,6 +254,13 @@ def pod_full_name
     "#{options[:pod_namespace]}/#{options[:pod_name]}"
   end
 
+  def inspector_admin_secret
+    kubeclient = kubernetes_client
+    sas = kubeclient.get_service_accounts
+    inspector_sa = sas.select { |sa| sa.metadata.name.include?(IMAGE_INSPECTOR_SA) }[0]
+    inspector_sa ? inspector_sa.imagePullSecrets[0].name : ""
```

Comment:
@enoodle add a `FIXME` or `TODO` comment saying that there could be multiple `imagePullSecrets` and we'll need to support that (but we need image-inspector to support it first).